src/python/WMArchive/Schemas/FWJRProduction.py

```diff
@@ -74,42 +73,54 @@
                          'runs': [{'lumis': [164, 165],
                                    'runNumber': 160960}],
                          'size': 647376,
-                         #TODO remove this 
-                         #'user_dn': '',
-                         #'user_vogroup': 'DEFAULT',
-                         #'user_vorole': 'DEFAULT',
                          'validStatus': 'PRODUCTION',
                          "SEName": "srm-cms.cern.ch",
-                         "PNN": "T2_CERN_CH",
+                         "PNN": "T2_CH_CERN",
                          "GUID": '',
                          "StageOutCommand": "srmv2-lcg"}],
-              'performance': {#"multicore": {},
-                  "storage": {
-                    "readAveragekB": 77.8474891246,
-                    "readCachePercentageOps": 0.0,
-                    'readMBSec': 0.0438598972596,
-                    'readMaxMSec': 4832.84,
-                    'readNumOps': 97620.0,
-                    'readPercentageOps': 1.00032780168,
-                    'readTotalMB': 7423.792,    
-                    'readTotalSecs': 0.0,
-                    'writeTotalMB': 357.624,
-                    'writeTotalSecs': 575158.0},
-                    "memory": {
-                       "PeakValueRss": 0.0,
-                       "PeakValueVsize": 0.0
-                    },
-                   "cpu": {
-                       "TotalJobCPU": 0.39894,
-                       "AvgEventCPU": -2.0, # for ("-nan")
-                       "MaxEventCPU": 0.0,
-                       "AvgEventTime": -1.0, # for ("inf")
-                       "MinEventCPU": 0.0,
-                       "TotalEventCPU": 0.0,
-                       "TotalJobTime": 26.4577,
-                       "MinEventTime": 0.0,
-                       "MaxEventTime": 0.0
-                   }},
+              'performance': {'cpu': [{'pName': 'TotalJobCPU',
```

Comment:
Seangchan, for schema definition we don't need actual values of pName, so I think we can just simplify to
"cpu":[{"pName":"bla", "value":0.1}] that's it. It instructs that cpu is a list of dicts where each dict item would have "pName" and "value" keys and "pName" value is string and "value" value is float.